src/content/overview/developer-portal/_index.md

```diff
@@ -6,7 +6,27 @@ menu:
   principal:
     parent: overview
     identifier: overview-developer-portal
-last_review_date: 2024-03-18
+last_review_date: 2024-06-11
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-product
 ---
+
+The Giant Swarm Developer Portal is the central self-service portal for your platform engineers and developers. It provides access to all the tools and resources needed to manage and develop applications on your Giant Swarm platform.
+
+Giant Swarm uses [Backstage](https://www.cncf.io/projects/backstage/) as the foundation for the developer portal. Backstage is an open-source platform for building developer portals. It provides a unified view of all your software development tools and services, navigate across the documentation even creating custom plugins to integrate with your existing tools.
```

Comment:
Maybe mention that it is open-source like everything else in our platform. Btw I took a look and we don't have any web or docs page explaining that our product is fully open source and there is no commercial license for it. Imo this would be good to have and then link to from pages like this.

Reply:
> Btw I took a look and we don't have any web or docs page explaining that our product is fully open source and there is no commercial license for it

IMO, the [intro page](https://github.com/giantswarm/docs/blob/main/src/content/overview/introduction/_index.md) is the perfect spot where to add it. I do an addition to it

Reply:
https://github.com/giantswarm/docs/pull/2232